src/Xhr.js

```diff
@@ -0,0 +1,23 @@
+import Cookies from 'js-cookie';
+
+export default class Xhr {
+    createSequenceAsset(spineVidId, courseId, projectId) {
+        const csrftoken = Cookies.get('csrftoken');
+        fetch('/sequence/api/assets/', {
```

Comment:
As far as I can think of, they have to be hardcoded somewhere. If the endpoint changes in Django, we'll have to update it in JavaScript as well. We can improve this situation by at least putting all the hard-coded endpoints in one place, taking cues from [Ember's router](https://guides.emberjs.com/v2.5.0/routing/defining-your-routes/#toc_basic-routes).